Shared/API/Onboarding+Cloud.swift

```diff
@@ -63,6 +63,10 @@ struct VerifyCode: CloudFunction {
            let token = dict["sessionToken"],
             !token.isEmpty {
             return dict
+        } else if let token = result as? String {
```

Comment:
Sometimes this comes back as a string. Until the backend fixes this, Im putting this here.